salesagility/SuiteCRM
# All menu - Display module by alphabetical order

## Description

All menu - Display module by alphabetical order
The admin "All" menu displays some entries to modules and their
associated modules as in "Products" and  "Product categories"
In english this allows they are listed next to each other, ordered by
the letter "P".
But when translating many language rotate the words on sentences, so it
turns into: "Produtos" and  "Categorias de Produtos" (example for Portuguese but its
the same on some other languages)... so they are placed in different
lines (here ordered by letters P and C)
This makes it difficult to browse for some modules with different
entries (as the Maps module)
Making a hard separation from the module name (products / maps) and the
dependent action (categories / cache / markers) makes it easy to
translators to have different translation form any similar wording from
the values on other language files!

**Note: this change is not a real alphabetical ordering (the feature should be available!) mas a workaround that allows each dependent modules be listed together**
## Motivation and Context
Easy list and navigation on the  "All menu"

## How To Test This

Apply changes in the "include" language file, reload!
See pic:
![5](https://cloud.githubusercontent.com/assets/3055443/17378887/c15895e8-59b7-11e6-82b6-04d19c98408c.png)

Same in selecting modules to menus (see Mapas)
![2017-09-20 18_53_42-microsoft edge](https://user-images.githubusercontent.com/3055443/30661242-4de0ec5e-9e3b-11e7-9abf-502bd8b2b572.png)



## Types of changes
  functionality to change)
### Final checklist
- [x] My code follows the code style of this project found
  [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [x] I have read the [**How to
  Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM)
  guidelines.
